library/std/diagnostics.qs

```diff
@@ -4,10 +4,71 @@
 namespace Microsoft.Quantum.Diagnostics {
     open QIR.Intrinsic;
 
+    /// # Summary
+    /// Dumps the current target machine's status.
+    ///
+    /// # Description
+    /// This method allows you to dump information about the current quantum state.
+    /// The actual information generated and the semantics are specific to each target machine.
+    ///
+    /// For the local sparse-state simulator distributed as part of the
+    /// Quantum Development Kit, this method will write the wave function as a
+    /// one-dimensional array of pairs of state indices and complex numbers, in which each element represents
+    /// the amplitudes of the probability of measuring the corresponding state.
+    ///
+    /// # Example
+    /// When run on the sparse-state simulator, the following snippet dumps
+    /// the Bell state $(\ket{00} + \ket{11}) / \sqrt{2}$ to the console:
```

Comment:
Didn't we decide to limit LaTeX to the 'Remarks' section? (CC @ScottCarda-MS)

Reply:
Yes we should avoid LaTex except in the Remarks section. In this case, we can use the ket unicode character: `〉` and the sqrt unicode character: `√` although we've also just referred to that as `sqrt(2)` in other places.